index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Jauhari Alafi</title>
+    <!-- Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <!-- CSS -->
+    <link rel="stylesheet" href="styles/global.css" />
+    <link rel="stylesheet" href="styles/template.css" />
+    <link rel="stylesheet" href="styles/main.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="nav">
+        <a href="#" class="nav-logo">Welcome</a>
+        <a href="#" class="nav-toggle"
+          ><img src="images/icons/hamburger-menu.svg" alt="Menu"
+        /></a>
+      </nav>
+    </header>
+    <section class="headline">
+      <div class="headline-primary">
+        <h2>
+          <span class="dark-blue">Hey there. I'm Mariane</span>
+          <br />
+          <span class="green">I'm a software developer</span>
+        </h2>
+      </div>
+      <div class="headline-sub">
+        <p class="dark-blue">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hestiate to contact me.
+        </p>
+      </div>
+      <div class="headline-socials">

```

Comment:
- [ ] Good job on the design, But for this project you are supposed to stick to the [Figma design ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)as much as possible. Please increase the gap between your social media icons and make sure it matches the design on the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/158779158-d88b6511-e11d-486f-b0a5-1bad80266ed5.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/158779351-23dd374a-acb5-4840-a1ba-19e45bb732ff.png)

Reply:
Hopefully, this is good enough?

![image](https://user-images.githubusercontent.com/67834048/158786023-de99b8bf-1740-4907-a4d4-5259ebaea4ab.png)